Add ISP provider management basic functions

pbishop16 opened the pull request:
This resolves #8 , including

added Country model
added Country seeder to load ISP countries

To load the countries please run php artisan db:seed --class=T4gCountryTableSeeder

Privileges (currently are in a very basic implementation; will create another issue to improve implementation)

I removed the country column from the ISP table due to my shortcomings in having the name displayed versus the country id.

@Dagolin Sorry about the delay on this issue

Dagolin commented on resources/views/isps/partials/form.blade.php:
```diff
@@ -0,0 +1,59 @@
+    {{ Form::text('executive_contact_skype', null, ['class' => 'form-control', 'placeholder' => ''])}}
+  </div>
+  <div class="form-group">
+    {{ Form::label('executive_contact_whatsapp', 'executive_contact_whatsapp:') }}
```
Should be `{{ Form::label('executive_contact_whatsapp', 'Executive Contact Whatsapp:') }}`

Dagolin commented:
@pbishop16 @ericfzeng I think we can leave out `show` operation since `edit` is enough, what do you think of it?

Dagolin commented:
Arr I am sorry, I take back my word. `show` provides friendly design that `edit` can't. Let's keep it

Dagolin commented on resources/views/isps/show.blade.php:
```diff
@@ -0,0 +1,96 @@
+
+                  <a href="{{ route('isps.edit', $isp->id) }}"
+                    class="btn btn-primary"
+                    role="button">Edit</a>
```
`staff` user should be able to `edit` since they can `add` record, and this button should be out of `delete form` block since it has nothing to do with `DELETE`

Dagolin commented:
@pbishop16 I have done my review, have to say it is a high quality PR and very impressive. The way you implement 'static::updating' event is my favorite! That's really something new to me. :+1:  impressive.

pbishop16 commented:
@Dagolin Greatly appreciated. Your customer management code really put me on the right path. I am learning a great deal from you. 

I fixed the noted errors. 

I also fixed an error that returned a 404page when trying to view the show page. The error occurred when the isp modifed_by was a null value. The solution is a bit hacky, but it resolves the error.

Please let me know if you have any concerns and/or suggestions.

Thank you.

Dagolin commented:
:+1:
